dags/utils/slack.py

```diff
@@ -0,0 +1,22 @@
+from airflow.models import Variable
+from airflow.providers.slack.operators.slack import SlackAPIPostOperator
+
+SLACK_CHANNEL = "#airflow-alerts"
```

Comment:
This looks like the recommended way of interfacing with slack. I think this channel should be configurable in case specific job owners want to direct their slack notifications to a non-default channel.

Reply:
hmm unfortunately when using a callback func it only expects a single argument for the context. We could change this to pull from Variables

Reply:
ill leave this as a future decision